src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -182,7 +182,10 @@ public static partial class RoslynInsertionTool
                     {
                         return;
                     }
```

Comment:
Should there be a `retainBuild = true;` here?

Reply:
nope there is a `!` in front.  If we get here then file copy failed and we should bail out